SimPPS/PPSPixelDigiProducer/interface/RPixChargeShare.h

```diff
@@ -0,0 +1,35 @@
+#ifndef SimPPS_PPSPixelDigiProducer_RPix_Charge_Share_H
+#define SimPPS_PPSPixelDigiProducer_RPix_Charge_Share_H
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include <vector>
+#include "Geometry/VeryForwardGeometry/interface/CTPPSPixelSimTopology.h"
+#include "SimPPS/PPSPixelDigiProducer/interface/RPixSignalPoint.h"
+
+class RPixChargeShare
+{
+public:
+  RPixChargeShare(const edm::ParameterSet &params, uint32_t det_id);
+  std::map<unsigned short, double, std::less<unsigned short> > Share(const std::vector<RPixSignalPoint> &charge_map);
```

Comment:
Isn't the `std::less<unsigned short>` redundant?

Reply:
It is not redundant; if removed, the code crashes at compilation time.

Reply:
Could you post the compilation error (here fully or a link elsewhere if it is long)? The third template argument should default to `std::less<unsigned short>` and thus should not be needed to be spelled out explicitly, so I wonder what is going on.

Reply:
I went back into the code myself and made a deeper check on all dependencies, what might have happened is that some occurrence was left behind. I'm really sorry about it. I'll issue an update as soon as other issues are solved (the code with the random generator, I just received input from someone who knows the code in details, I'll comment further in the corresponding message). Once again, sorry about this.